### DISCUSSION TITLE
Admin panel

### DISCUSSION THREAD

**Discussion by Chewbaka69** [lanedirt/OGameX#112] [Ideas]
Hi,

I can be nice, to maybe add an admin panel, to configure the speed of the server, like this following info:
- Default build queue
- Premium build queue
- Default research queue
- Premium research queue
- Fleet speed: x1, x2, x3, x4, x5, x6
- Economy speed: x1, x2, x3, x4, x5, x6, x7
- Galaxies: 4*,5, 6, 7, 8, 9 (9= normal/standard).
- Debries Field: 30%, 40%, 50%, 60%, 70%, 80% (the standard value is 30%).
- DiDF- Defense into debris: 0, 30% 40% 50% 60%, 70% 80% (the standard is 0)
- Deuterium Consumption: 0.5, 0.6, 0.7, 0.8, 0.9, 1 (the default value is 1).
- Initial Dark Matter: 8.000 - 25.000
- Planet Fields: +10,+25 or +30 fields can be requested (standard value is 0).
- Research speed: x2, x4, x6, x8, x10, x12, x14 (multiplicator of the Eco speed) (standard value is 1).
- Espionage probe capacity: ON/OFF (OFF is de default setting).

And to monitor some other stuff like:
- manage univers (number of planet, number of system, default planet size)
- multi account
- fleet in fly
- Edit planet / moon / user
- ban / block
- logs (constructions, fleet send, etc.)
- manage crontask

Just some ideas for the admin panel :)
But well done and good job, because it looks like the true.

**Comment by lanedirt** [comment]
Hi there, thanks for your suggestion. Adding an admin panel sounds like a good idea. Thanks for the overview of the possible server settings.

Currently there's still some work to be done on creating all the basic game features. Once all the basics are in I will start focusing on the admin panel system.

Thanks for your ideas, it's appreciated!

Edit: I created a new issue #115 for this feature.

**Reply by Chewbaka69** [reply]
Yes sure :) 👍
